runatlantis.io/docs/server-side-repo-config.md

```diff
@@ -78,10 +78,6 @@ repos:
   post_workflow_hooks: 
     - run: my-post-workflow-hook-command arg1
 
-  # api-secret defines the secret used to pass to the /api/plan and /api/apply endpoints
-  # if not present the api endpoints will not be accesible.
-  api-secret: "myapisecret"
```

Comment:
Why remove this from here?

Reply:
If I'm not mistaken, I don't think `api-secret` works as a server side repo config (for example, it's not defined in the [`Repo`](https://github.com/runatlantis/atlantis/blob/main/server/core/config/raw/global_cfg.go#L22) struct like the other configs are). Removing the section to avoid confusion.

Reply:
it is defined in the server side repo config, I have tested it ( I added that part of the doc)

Reply:
@jamengual Hmm... is it possible that you have passed `ATLANTIS_API_SECRET` environment variable when you tested it? If  `api-secret` is not defined as part of the [`Repo`](https://github.com/runatlantis/atlantis/blob/main/server/core/config/raw/global_cfg.go#L22) struct, Atlantis cannot parse the value, right? 🤔